### PR TITLE
Adding autoesda Recipe

### PR DESCRIPTION
**Pull request by NicholasDeKock**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by NicholasDeKock** [comment]
@conda-forge/help-python The recipe has passed all checks and is ready for your review. Thanks

**Comment by NicholasDeKock** [comment]
<code>@</code>`conda-forge-admin The recipe has passed all checks and is ready for your review. Thanks

**Comment by NicholasDeKock** [comment]
@conda-forge/help-python The recipe has passed all checks and is ready for your review. Thanks

File: LICENSE.txt [hunk 1]
```diff
@@ -1,27 +1,30 @@
-Copyright (c) 2015-2022, conda-forge
```
Comment (synapticarbors) [inline LICENSE.txt:1]: Please do not modify this file as it is specific to conda-forge and not this package.

File: recipes/autoesda/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - esda
+    - geopandas
+    - libpysal
+    - matplotlib
```
Comment (synapticarbors) [inline recipes/autoesda/meta.yaml:25]: If you're not using the Qt features of `matplotlib`, you could change this to `matplotlib-base`.

File: recipes/autoesda/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
```
Comment (synapticarbors) [inline recipes/autoesda/meta.yaml:21]: The upstream package pins the versions of many the dependencies to a specific version. I just wanted to double check that having no version restrictions is known to be ok.

**Review by synapticarbors — CHANGES_REQUESTED** [review]

**Review by NicholasDeKock — COMMENTED** [review]
Thank you for the changes, I am happy for them to be made - Is there anything more I need to do on my side?

File: recipes/autoesda/meta.yaml (hunk 2 above)
Comment (NicholasDeKock) [reply recipes/autoesda/meta.yaml:25]: Unfortunately the library makes use of the Qt features and Qt is listed as a dependancy of some of the autoesda dependancies

File: recipes/autoesda/meta.yaml (hunk 3 above)
Comment (NicholasDeKock) [reply recipes/autoesda/meta.yaml:21]: This is fine

**Review by synapticarbors — APPROVED** [review]

**Pull request merged**